src/common/components/headBlock/headBlockDetail.tsx

```diff
@@ -105,19 +105,19 @@ const HeadBlockDetail = (props:any) => {
                             // <StringField key={index} value={result[key]}  item={key} number={index} label_for='block'/>
                             <>
                             {typeof(result[key])!=="object"?
-                            <StringField key={index} value={result[key]}  item={key} number={index} label_for='block'/>
+                            <StringField key={index+Math.floor(Math.random() * 3000)+2000} value={result[key]}  item={key} number={index} label_for='block'/>
```

Comment:
@ahmedaftab-RT Please don't use math.random for keys, instead fine unique id or hash from transaction/blocks/lists to use instead of math.random, for all instances and keys